components/screens/Dashboard.tsx

```diff
@@ -77,7 +77,7 @@ export default function Dashboard() {
           </div>
 
           <span className="text-sm text-daintree-400">
-            ✋👑 Hold on, Your Majesty! Please back up your recovery phrase 📜.
+            ✋👑 Hold on, Your Majesty! Please back up you r recovery phrase 📜.
```

Comment:
```suggestion
            ✋👑 Hold on, Your Majesty! Please back up your recovery phrase 📜.
```
It seems typos here.